### PR TITLE
Umer mongodb

### PR DESCRIPTION
**Pull request by uzher-code**
Added connection uri to .env and connected mongo db to the server.

Also added .env to gitignore.

Sent the database setup info doc to team on slack

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,4 +20,4 @@ yarn-error.log*
 
 
 #TODO add .env to your gitignore
-
+server/.env
```
Comment (dproc96) [inline .gitignore:23]: It doesn't look like this worked as your env file is still appearing in the diff, did you git remove the file in addition to adding it here?

**Review by dproc96 — APPROVED** [review]
This is working and looks good, but double check on removing the .env file. You will also need to change the DB password as it is exposed in the commit history now FYI

**Comment by bonnieli** [comment]
1. Resolve conflict
2. Make sure you clear .gitignore cache 

Success looks like if you view files changed, you won't be able to see your .env file

File: .gitignore (hunk 1 above)
Comment (uzher-code) [reply .gitignore:23]: cleared cache. gitignore is now fixed

**Pull request merged**